store/copr/coprocessor.go

```diff
@@ -97,6 +97,14 @@ func (c *CopClient) Send(ctx context.Context, req *kv.Request, variables interfa
 	bo := backoff.NewBackofferWithVars(ctx, copBuildTaskMaxBackoff, vars)
 	ranges := NewKeyRanges(req.KeyRanges)
 	tasks, err := buildCopTasks(bo, c.store.GetRegionCache(), ranges, req, eventCb)
+	reqType := "null"
+	if req.ClosestReplicaReadAdjuster != nil {
+		reqType = "miss"
+		if req.ClosestReplicaReadAdjuster(req, len(tasks)) {
+			reqType = "hit"
```

Comment:
I this Yes/No may be better than hit/miss. We often use hit/miss on the data cache.

Reply:
em.. Yes/No sounds a bit odd to me